ir/src/values/mod.rs

```diff
@@ -41,6 +41,25 @@ pub enum Integer {
     I128(i128),
 }
 
+impl TryFrom<Integer> for u32 {
```

Comment:
@0rphon @Protryon who is calling this function?
I do not see it being used in snarkVM currently.

Reply:
line 49
https://github.com/AleoHQ/leo/blob/ir/compiler/src/statement/assign/assignee/array_range_index.rs